pytorch_lightning/accelerators/ipu.py

```diff
@@ -31,3 +32,7 @@ def auto_device_count() -> int:
         # TODO (@kaushikb11): 4 is the minimal unit they are shipped in.
         # Update this when api is exposed by the Graphcore team.
         return 4
+
+    @staticmethod
+    def is_available() -> bool:
+        return _IPU_AVAILABLE
```

Comment:
You don't need to do this here but I think it would be nicer to move the _IPU_AVAILABLE implementation here and remove this global variable.

Same comment for the other classes

Reply:
I agree, I don't think the IPU available and TPU available make sense under imports. That's a good follow up for a separate PR

Reply:
@carmocca there's also a whole XLADeviceUtils which probably can be deduped further for TPUs @kaushikb11 https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/xla_device.py